### PR TITLE
Feature/custom checkbox

### PR DESCRIPTION
**Pull request by gustavomelki**
I rewritten the checkbox component created in data table using the template tag. I changed too the new checkbox in data table.

### CODE REVIEW

File: src/components/Checkbox/checkbox.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        
+        .sb-checkbox__truthy {
+          stroke-dashoffset: 0;
+          transition: stroke-dashoffset .2s cubic-bezier(.4,0,.6,1) 0ms;
+        }
+      }
+
+      &[disabled] {
+        & + .sb-checkbox__input .sb-checkbox__truthy {
+          stroke: #b1b5be;
```
Comment (ademarCardoso) [inline src/components/Checkbox/checkbox.scss:46]: Change the `stroke: #b1b5be;` to `stroke: $light-gray;`

File: src/components/Checkbox/checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        
+        .sb-checkbox__indet {
+          transform: rotate(0) scale(1);
+          transition: transform $checkbox-transition;
+        }
+      }
+
+      &[disabled] {
+        & + .sb-checkbox__input .sb-checkbox__indet {
+          fill: #b1b5be;
```
Comment (ademarCardoso) [inline src/components/Checkbox/checkbox.scss:71]: Same thing, change to `$light-gray`.

File: src/components/Checkbox/checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        }
+      }
+    }
+
+    &[disabled] {
+      & + .sb-checkbox__input {
+        border: solid 0.1rem $light;
+        background-color: #f5f5f5;
```
Comment (ademarCardoso) [inline src/components/Checkbox/checkbox.scss:79]: Change the `background-color` to: `$white`;

**Review by ademarCardoso — COMMENTED** [review]
It was great my friend 👍

**Comment by emanuelgsouza** [comment]
@gustavomelki , I don't have any comments. When you resolve the Ademar's points, we can merge this PR.

So, that's a great work.

File: src/components/Checkbox/checkbox.scss (hunk 1 above)
Comment (gustavomelki) [reply src/components/Checkbox/checkbox.scss:46]: Thanks!

File: src/components/Checkbox/checkbox.scss (hunk 3 above)
Comment (gustavomelki) [reply src/components/Checkbox/checkbox.scss:79]: Disabled checkbox background isn't white!

**Pull request merged**